circup/__init__.py

```diff
@@ -1211,11 +1650,21 @@ def tags_data_save_tag(key, tag):
     message="%(prog)s, A CircuitPython module updater. Version %(version)s",
 )
 @click.pass_context
-def main(ctx, verbose, path, board_id, cpy_version):  # pragma: no cover
+def main(ctx, verbose, path, host, password, board_id, cpy_version):  # pragma: no cover
+    # pylint: disable=too-many-arguments
     """
     A tool to manage and update libraries on a CircuitPython device.
     """
     ctx.ensure_object(dict)
+
+    using_webworkflow = "host" in ctx.params.keys() and ctx.params["host"] is not None
+
+    ctx.obj["using_webworkflow"] = using_webworkflow
```

Comment:
Do you still need this?

Reply:
Nope, with a bit more shifting bits around in a few other places I was able to remove the need for it.